app/controllers/api/v1/users_controller.rb

```diff
@@ -2,19 +2,24 @@
 
 module Api
   module V1
-    # Register users
-    class UsersController < ApplicationController
+    # Manages users
+    class UsersController < AuthenticatedController
+      def show
+        render json: current_user
+      end
```

Comment:
Shall we show not only current user? I'd like to separate `current_user` and `user` logic

Reply:
Do you suggest to move it to a separate controller?

Reply:
Yep